app/screens/TransactionAuthorization.tsx

```diff
@@ -196,34 +193,14 @@ export function TransactionAuthorization (): JSX.Element | null {
     }
   }, [transaction, wallet, status, authentication, attemptsRemaining])
 
-  // Disable Biometric hook
-  /* // setup biometric hook if enrolled
-  useEffect(() => {
-    BiometricProtectedPasscode.isEnrolled()
-      .then(isEnrolled => setIsBiometric(isEnrolled))
-      .catch(e => Logging.error(e))
-  }, [wallet]) */
-
-  // prompt biometric auth in 'PIN' event
+  // auto resolve promptPassphrase promise using cached pin if available
   useEffect(() => {
     if (status === 'PIN' && pin.length === PIN_LENGTH) {
       onPinInput(pin)
     }
+  }, [status, pin])
 
-    // biometric disabled
-    // if (status === 'PIN' && isBiometric) {
-    //   LocalAuthentication.authenticateAsync()
-    //     .then(async () => await BiometricProtectedPasscode.get())
-    //     .then(pinFromSecureStore => {
-    //       if (pinFromSecureStore !== null) {
-    //         onPinInput(pinFromSecureStore)
-    //       }
-    //     })
-    //     .catch(e => Logging.error(e)) // auto fallback to manual pin input
-    // }
-  }, [status, pin/*, isBiometric */])
-
-  if (status === 'INIT' || status === 'IDLE') {
+  if (status === 'INIT' || status === 'IDLE' || status === 'BLOCK') {
```

Comment:
Tested on android, TransactionAuthorization will no longer show until the PIN event emitted.